src/partials/contacts.html

```diff
@@ -1,48 +1,56 @@
 <section class="contacts">
   <div class="container">
-    <h2 class="contacts__header">Contacts</h2>
+    <h2 class="contacts__header--main">Contacts</h2>
```

Comment:
Jak używamy mixins musimy pamiętać, aby wyglądało to np tak "contacts__header contacts__header--main", napisanie samego "contacts___header--main" niesetety nie jest wystarczające. Mixin nie może istnieć bez bazy. Proszę zatem o zastosowanie tego wszędzie gdzie jest to konieczne :)

Reply:
uhh faktycznie, już podmieniam, bo tu się wykrzaczyło wszystko xD